plenum/common/messages/node_messages.py

```diff
@@ -173,7 +174,10 @@ class Commit(MessageBase):
                                                 optional=True)),
         # PLUGIN_FIELDS is not used in Commit as of now but adding for
         # consistency
-        (f.PLUGIN_FIELDS.nm, AnyMapField(optional=True, nullable=True))
+        (f.PLUGIN_FIELDS.nm, AnyMapField(optional=True, nullable=True)),
```

Comment:
I believe we don't use PLUGIN_FIELDS at all now, so maybe we can add `BLS_SIGS` before `PLUGIN_FIELDS`?